src/scripts/main.js

```diff
@@ -1,7 +1,25 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const elem = document.createElement('div');
+
+  document.body.prepend(elem);
+
+  elem.className = `notification ${type}`;
+  elem.innerHTML = `<p>${description}</p>`;
+  elem.style.top = posTop + 'px';
+  elem.style.right = posRight + 'px';
+
+  const heading = document.createElement('h2');
+
+  document.querySelector('.notification').prepend(heading);
+
+  heading.className = 'title';
+  heading.textContent = `${title}`;
+
+  setTimeout(() => {
+    elem.style.display = 'none';
+  }, 2000);
```

Comment:
The `setTimeout` duration of 2000 milliseconds (2 seconds) may be too short for users to read the notification. Consider increasing the duration or making it configurable.